README.md

```diff
@@ -14,10 +14,12 @@
 </br>
 
 <div align="center">
+  <a href="https://novu.co">
```

Comment:
What do you think of doing target _blank?

Reply:
> What do you think of doing target _blank?

You are absolutely right.
Done